ceph-salt.8

```diff
@@ -0,0 +1,213 @@
+\" Man page for ceph-salt
+.TH "ceph-salt" "8" "" "" "ceph-salt"
+.SH "NAME"
+ceph\-salt \- deploy ceph clusters using cephadm
+.SH "SYNOPSIS"
+.sp
+\fBceph-salt\fP [\fI\-\-global\-opts\fP] \fIcommand\fP
```

Comment:
The synopsis mentions `[--global-opts]`, but these are not explained/listed anywhere.

Reply:
Fixed. I've now added a section for **GLOBAL OPTIONS**.